addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -306,60 +312,159 @@ private synchronized void startAutomaticRefresh() {
         if (pollingJob == null || pollingJob.isCancelled()) {
             pollingJob = scheduler.scheduleWithFixedDelay(pollingRunnable, 0, refreshInterval, TimeUnit.SECONDS);
         }
-        if (sendCommandJob == null || sendCommandJob.isCancelled()) {
-            sendCommandJob = scheduler.scheduleWithFixedDelay(sendCommandRunnable, 0, sendCommandInterval,
-                    TimeUnit.SECONDS);
+        if (queueConsumerThread == null || !queueConsumerThread.isAlive()) {
```

Comment:
The scheduler used a polling method, waking up a Thread (cpu) and caused a noticeable delay. The `queueConsumerThread` will wait() on reading the queue.